### DISCUSSION TITLE
[HW4] Is it okay to use Trainer API in huggingface? SQuAD version?

### DISCUSSION THREAD

**Discussion by haneul-yoo** [seominjoon/kaist-ai605#57] [Q&A] [answered]
Hello!

1. There is a tutorial link for problem 2 and 3 which fine-tune a pretrained model in native PyTorch.
Is it okay to use the Trainer API in PyTorch for fine-tuning to sst-2 and squad?

2. Which dataset should we use for problem 3: SQuAD 1.0 or SQuAD 2.0? Or doesn't matter?

Regards,

**Comment by seominjoon** [comment, marked as answer]
1. Yes, that's fine to use Trainer. The tutorial link was to native PyTorch so that you can better understand it.
2. Please use SQuAD 1.0 (which is easier).
